packages/edit-navigation/src/components/menus-editor/index.js

```diff
@@ -11,41 +16,72 @@ import {
 	SelectControl,
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+const { DOMParser } = window;
 
 /**
  * Internal dependencies
  */
 import CreateMenuArea from './create-menu-area';
 import NavigationEditor from '../navigation-editor';
 
-export default function MenusEditor( { blockEditorSettings } ) {
-	const { menus, hasLoadedMenus } = useSelect( ( select ) => {
-		const { getMenus, hasFinishedResolution } = select( 'core' );
-		const query = { per_page: -1 };
-		return {
-			menus: getMenus( query ),
-			hasLoadedMenus: hasFinishedResolution( 'getMenus', [ query ] ),
-		};
-	}, [] );
+const noticeId = uniqueId(
+	'navigation-editor/menu-editor/edit-navigation-delete-menu-error'
```

Comment:
This still means that all instances of the component will share the same ID. This should be in a ref.